package.json

```
@@ -0,0 +1,65 @@
+{
+    "name": "react-native-cli-templates",
+    "version": "1.0.0",
```

0.0.0 is used to indicate that this is not a published package
```suggestion
    "version": "0.0.0",
```